controllers/datadogagent/controller_test.go

```diff
@@ -736,7 +736,7 @@ func TestReconcileDatadogAgent_Reconcile(t *testing.T) {
 			wantErr: false,
 			wantFunc: func(c client.Client) error {
 				configmap := &corev1.ConfigMap{}
-				if err := c.Get(context.TODO(), newRequest(resourcesNamespace, getSystemProbeConfigConfigMapName(resourcesName)).NamespacedName, configmap); err != nil {
+				if err := c.Get(context.TODO(), newRequest(resourcesNamespace, fmt.Sprintf("%s-%s", resourcesName, SystemProbeConfigMapSuffixName)).NamespacedName, configmap); err != nil {
```

Comment:
I think you can still use [getSystemProbeConfigConfigMapName()](https://github.com/DataDog/datadog-operator/blob/8b4b7966c329638b5c7f4fbf9f5a6e6fbc7495c9/controllers/datadogagent/systemprobe.go#L323)

Reply:
The `dda` object is unfortunately not accessible from this method.